### PR TITLE
ui: collect statement stats from all nodes

### PR DESCRIPTION
**Pull request by couchand**
The initial statement stats API endpoint only exposed the local node's statements.  This extends it to query all the other nodes as well, properly aggregates across nodes, and provides the per-node details on the statement details page.

screen shot 2018-06-28 at 5 19 00 pm

### CODE REVIEW

**Comment by couchand** [comment]
I think this is ready to go @vilterp PTAL

**Comment by couchand** [comment]
Oh I still need to aggregate the results...

**Comment by couchand** [comment]
Probably still want to add a proper test for this, but it's working!

File: pkg/server/serverpb/status.proto [hunk 1]
```diff
@@ -608,6 +608,7 @@ message StoresResponse {
 }
 
 message QueriesRequest {
+  string node_id = 3 [ (gogoproto.customname) = "NodeID" ];
```
Comment (vilterp) [inline pkg/server/serverpb/status.proto]: nit: no spaces inside `[ ]`s

**Review by vilterp — COMMENTED** [review]
Looks good; add a test and :shipit:

**Comment by couchand** [comment]
This has now been rebased on top of both #26949 and #26821, only the last six commits should be reviewed here.  This should be ready for another look @vilterp, maybe @knz since I made use of your refactor, and anyone else who would like to weigh in here.

**Comment by vilterp** [comment]
Noice! 
image

Layout suggestion to save a bit of width: show just the node ids ("n1", etc) and put the host and port in a tooltip? Will be esp. helpful when hostnames are longer, which they often are (`cockroach.my-kubernetes-cluster.my-co.internal`...)

Also, nit: a couple of pixels under the h3s (between them and the tables) would be nice.

**Comment by knz** [comment]
I would recommend holding on merging this until #26821 merges, since Ben made a recommendation about a tweak in the API and I still need to process that recommendation.

File: pkg/server/status_test.go [hunk 2]
```diff
@@ -849,3 +852,72 @@ func TestRemoteDebugModeSetting(t *testing.T) {
 		}
 	}
 }
+
+func TestStatusAPIQueries(t *testing.T) {
```
Comment (vilterp) [inline pkg/server/status_test.go]: I guess this should be called `TestStatusAPIStatements`? (Thx for doing the rename btw)

**Review by vilterp — APPROVED** [review]
LGTM with a minor nit. Tested locally; code looks good.

**Comment by couchand** [comment]
Rebased on master, last few comments addressed, and ready to go when I see a green build I think.

**Comment by couchand** [comment]
ok, ok let's just go

bors r+

**Pull request merged**